geoips/plugins/yaml/workflows/visir/Infrared.yaml

```diff
@@ -0,0 +1,31 @@
+interface: workflows
+family: order_based
+name: ob_Infrared
+docstring: |
+  The Infrared Workflow configuration.
+spec:
+  steps:
+    - interpolator:
+        name: interp_nearest
+        arguments: {}
+    - algorithm:
+        name: single_channel
+        arguments:
+          output_data_range: [-90.0, 30.0]
+          input_units: Kelvin
+          output_units: celsius
+          min_outbounds: crop
+          max_outbounds: crop
+          norm: null
+          inverse: null
+    - output_formatter:
+        name: imagery_annotated
+        arguments:
+          colormapper:
+            name: Infrared
+            arguments:
+              data_range: [-90.0, 30.0]
```

Comment:
We had talked about having this outside the output formatter potentially, and add some sort of metadata to the xarray object to later use within the output formatter plugin in determining how to plot ?  That would allow us to more easily pull the "output_formatter" part out into a separate workflow ?

Reply:
Yep, we're planning that in later PRs. For now, we're not really handling ColorMapper at all yet. The intent is to have ColorMapper plugins modify the metadata of our data object rather than supplying the ColorMapper directly to the OutputFormatter as an argument. That's just later 😄